packages/buckets/src/api/index.ts

```diff
@@ -118,6 +123,118 @@ export type PathObject = {
   root?: RootObject
 }
 
+/**
+ * ArchiveConfig is the desired state of a Cid in the Filecoin network.
+ */
+export interface ArchiveConfig {
+  /**
+   * RepFactor (ignored in Filecoin testnet) indicates the desired amount of active deals
```

Comment:
Marking some of these as ignored in the comments, but want to keep them in the code for forward compatibility.

Reply:
But you don't wanna mark them as optionals?

Reply:
No, because if someone is going to set a new default, they should set it to what they will want in the future, or be explicit about providing a value like 0 (which will not work in the future, but at least they know they set it that way)